user/details/style.css

```diff
@@ -278,3 +278,32 @@ footer p a {
     height: 20px;
   }
 }
+
+.accordion-disabled {
+  opacity: 0.5;
+  cursor: none;
+}
+
+.tool-parent {
+  position: relative;
+}
+
+.tooltip {
+  position: absolute;
+  bottom: 120%;
+  left: -120%;
```

Comment:
Can we please use +ve values here?
Try using `right` property

Reply:
same issue to using `right` property
```css
bottom: calc(100% + 10px);
right: calc(50% - 100px);
```
then used above code is it right ?